x-pack/plugins/canvas/public/transitions/rotate/index.js

```diff
@@ -5,11 +5,16 @@
  */
 
 import './rotate.css';
+import { i18n } from '@kbn/i18n';
 
 export const rotate = () => ({
   name: 'rotate',
-  displayName: 'Rotate',
-  help: 'Rotate from one page to the next',
+  displayName: i18n.translate('xpack.canvas.transitions.rotateDisplayName', {
```

Comment:
please add .rotate. namespace for ids in this file. Do the same for other files in this MR